models/User.go

```diff
@@ -1743,7 +1743,7 @@ func PerformOrderFillTransaction(ask *Ask, bid *Bid, stockTradePrice uint64, sto
 
 	/* We're here, so both orders are open now */
 
-	var updateDataStreams = func(askTrans, bidTrans, askTaxTrans, bidTaxTrans *Transaction) {
+	var updateDataStreams = func(askTrans, bidTrans, askTaxTrans, bidTaxTrans, askUpdateTrans, bidUpdateTrans *Transaction) {
```

Comment:
Can you rename it to something better? askReserveUpdateTrans makes more sense even though it's a bit lengthy.